nginx/datadog_checks/nginx/metrics.py

```diff
@@ -112,6 +112,7 @@
     'nginx.upstream.peers.unavail',
 ]
 
```

Comment:
Can we add a comment similar to https://github.com/DataDog/integrations-core/pull/11252/files#diff-4019c9169a867e9126b9829c0e636dfa73c201d4d856fc2c3b8f0314b7ef22eeR117?

Reply:
I'm not sure where the link is taking me to